src/app/shared/nav-bar/nav-bar.component.scss

```diff
@@ -0,0 +1,73 @@
+.navBar{
```

Comment:
css naming conversion doesn't use `CamelCase` but uses `Kebab case`.
`.navBar` is wrong, `.nav-bar` is right.